src/app/Services/UserService.php

```diff
@@ -95,4 +95,16 @@ public function getUserProfile(int $id): array
             return ['error' => 'An error occurred: ' . $e->getMessage()];
         }
     }
+
+    /**
+     * Get query
+     */
+    public function getQuery($request, int $id)
```

Comment:
parameter type of $request and return type as well

Reply:
Done